packages/backend/src/config/app.ts

```diff
@@ -18,10 +18,10 @@ function envFloat(name: string, value: number): number {
   return envValue == null ? value : +envValue
 }
 
-function envBool(name: string, value: boolean): boolean {
-  const envValue = process.env[name]
-  return envValue == null ? value : envValue === 'true'
-}
+// function envBool(name: string, value: boolean): boolean {
+//   const envValue = process.env[name]
+//   return envValue == null ? value : envValue === 'true'
+// }
```

Comment:
We can just remove this altogether (we can grab it from `auth` package, or just go back to the commits if we need it again)

Reply:
Sure, happy to remove it. I think it was commented out before we un-commented when introducing the bypass flag, that's why commented out again.